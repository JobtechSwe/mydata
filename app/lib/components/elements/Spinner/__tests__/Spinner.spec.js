import React from 'react'
import { shallow } from 'react-native-testing-library'
import { Spinner } from '../Spinner'

jest.useFakeTimers() // this is needed to make jest exit after test

describe('elements/Spinner', () => {
  it('should render', () => {
    const { output } = shallow(<Spinner />)

    expect(output).toMatchSnapshot()
  })
})
