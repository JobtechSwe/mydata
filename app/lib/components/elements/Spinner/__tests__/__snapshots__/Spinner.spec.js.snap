// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/Spinner should render 1`] = `
<ForwardRef
  style={
    Object {
      "transform": Array [
        Object {
          "rotate": "0deg",
        },
      ],
    }
  }
/>
`;
